projects/vdk-plugins/vdk-structlog/src/vdk/plugin/structlog/formatters.py

```diff
@@ -0,0 +1,93 @@
+# Copyright 2021-2023 VMware, Inc.
+# SPDX-License-Identifier: Apache-2.0
+import logging
+
+from pythonjsonlogger import jsonlogger
+from vdk.plugin.structlog.constants import CONSOLE_STRUCTLOG_LOGGING_METADATA_ALL
+from vdk.plugin.structlog.constants import RECORD_DEFAULT_FIELDS
+from vdk.plugin.structlog.constants import STRUCTLOG_LOGGING_METADATA_ALL
+from vdk.plugin.structlog.filters import ConsoleMetadataFilter
+from vdk.plugin.structlog.filters import JsonMetadataFilter
+
+
+class CustomJsonFormatter(jsonlogger.JsonFormatter):
```

Comment:
What does custom means ?  I think you should be able to come with a bit more explicit and clear name. 

Again a reminder ot add suitable pydoc to most classes and public methods